source/Magritte-Model/MACompositeAccessor.class.st

```diff
@@ -1,5 +1,5 @@
 "
-When you have a ""to-many"" relationship, I describe the flattened collection of its items - something like {{gtMethod:Collection>>#flatCollect:}} meets {{gtClass:MAChainAccessor}}. Here's a usage example {{gtExample:MACompositeElementBuilder class>>#example}}:
+When you have a ""to-many"" relationship, I describe the flattened collection of its items - something like {{gtMethod:Collection>>#flatCollect:}} meets {{gtClass:MAChainAccessor}}. Here's a usage example `MACompositeElementBuilder class>>#example`:
```

Comment:
We cannot keep the `{{gtExample:MACompositeElementBuilder class>>#example}}`, because we check all links on every release and all links must be valid. We do not have better solution to this case at this moment.